spec/system/admin/admin_manages_organizations_spec.rb

```diff
@@ -23,7 +23,7 @@
 
     it "does not show the remove form when there are no credits" do
       expect(page).to have_button("Add Org Credits")
-      # expect(page).to have_no_button("Remove Org Credits")
+      expect(page).to have_no_button("Remove Org Credits")
```

Comment:
no clue what triggered this being commented out in https://github.com/forem/forem/pull/7885 

If it's passing - looks good to me.